src/App.vue

```diff
@@ -673,37 +630,39 @@ export default class App extends Mixins(
       nrNumber: string
     }
     if (!nameRequest) {
-      throw new Error(`Invalid ${filingName} filing - Name Request object`)
+      throw new Error('Invalid bootstrap filing - missing name request object')
     }
 
     // verify that this is a supported entity type
     const legalType = nameRequest.legalType
     if (!legalType || !this.supportedEntityTypes.includes(legalType)) {
-      throw new Error(`Invalid ${filingName} filing - legal type`)
+      throw new Error(`Invalid bootstrap filing - missing or unsupported legal type = ${legalType}`)
     }
 
     // store business info
-    this.setEntityStatus(entityStatus)
+    this.setBootstrapFilingStatus(status)
+    this.setBootstrapFilingType(filingName)
```

Comment:
Now we save individual status and type instead of a combined type value (which was getting messy).